b2luigi/cli/process.py

```diff
@@ -5,7 +5,7 @@
 __has_run_already = False
 
 
-def process(task_like_elements, show_output=False, dry_run=False, test=False, batch=False, **kwargs):
+def process(task_like_elements, show_output=False, dry_run=False, test=False, batch=False, ignore_additional_command_line_args=False, **kwargs):
```

Comment:
I think you might also want to pass those argument to the called CLI method, or?

Reply:
Fixed, thanks! Somehow that hadn't made it into the commit.